server/copilot_client.py

```
@@ -94,9 +107,13 @@ async def stop(self) -> None:
             try:
                 if hasattr(self._sdk_client, "stop"):
                     await self._sdk_client.stop()
-            except Exception:
-                if hasattr(self._sdk_client, "force_stop"):
-                    await self._sdk_client.force_stop()
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)